packages/graphile-build-pg/src/omit.ts

```diff
@@ -75,8 +77,19 @@ function parse(arrOrNot, errorPrefix = "Error") {
 
 export default function omit(
   entity: PgProc | PgClass | PgAttribute | PgConstraint,
-  permission: string
-) {
+  permission:
+    | "create"
+    | "read"
+    | "update"
+    | "delete"
+    | "filter"
+    | "order"
+    | "all"
+    | "many"
+    | "execute"
+    | "base"
+    | string
```

Comment:
(I know the individual permissions above are redundant because of the `string`. I want to keep them anyway.)